lib/repository.rb

```diff
@@ -1,22 +1,21 @@
 require 'csv'
 require 'pry'
 
-
 class Repository
 
-  attr_reader :path, :data
+  attr_reader :path, :data, :klass
 
   def initialize(path, klass)
     @path = path
     @data = []
     @klass = klass
   end
 
-
   def load_file
     CSV.foreach @path, headers: true, header_converters: :symbol do |row|
       @data << @klass.new(row.to_hash)
     end
+    data
   end
```

Comment:
This is bad practice. 

You've already got `@data` as an attribute on your instance, so you just need to call `my_instance.data`